components/dashboard/src/projects/Projects.tsx

```diff
@@ -77,7 +77,18 @@ export default function () {
                 <h3 className="text-center text-gray-500 mt-8">No Recent Projects</h3>
                 <p className="text-center text-base text-gray-500 mt-4">Add projects to enable and manage Prebuilds.<br /><a className="gp-link" href="https://www.gitpod.io/docs/prebuilds/">Learn more about Prebuilds</a></p>
                 <div className="flex space-x-2 justify-center mt-7">
-                    <Link to={newProjectUrl}><button>New Project</button></Link>
+                    <Link to={newProjectUrl}><button onClick={() => {
+                        //track dashboard_button_clicked 'new_project'
+                        getGitpodService().server.trackEvent({
+                            event: "dashboard_button_clicked",
+                            properties:{
+                                dashboard_context: "/projects",
+                                button: "new_project",
+                                button_context: "primary_button"
+                            }
+                        })
+                    }
+                    }>New Project</button></Link>
                     {team && <Link to="./members"><button className="secondary">Invite Members</button></Link>}
```

Comment:
**suggestion**: There's also an _Invite Members_ button on the Projects list. Shall we also track this?

Reply:
Done!👍🏼